db/migrate/20190730162819_create_uphold_connection_for_channel.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUpholdConnectionForChannel < ActiveRecord::Migration[5.2]
+  def change
+    create_table :uphold_connection_for_channels, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.belongs_to :uphold_connection, type: :uuid, index: true, null: false
```

Comment:
Let's add uniqueness constraints just in-case

Reply:
Should not be unique on this, because users will have multiple channels